src/apps/vectorwar/gdi_renderer.cpp

```diff
@@ -168,7 +168,7 @@ GDIRenderer::DrawConnectState(HDC hdc, Ship &ship, PlayerConnectionInfo &info, C
 
       case Disconnecting:
          sprintf(status, "Waiting for player...");
-         progress = (timeGetTime() - info.disconnect_start) * 100 / info.disconnect_timeout;
+         progress = (Platform::GetCurrentTimeMS() - info.disconnect_start) * 100 / info.disconnect_timeout;
```

Comment:
The `Platform` type is an internal implementation detail of the GGPO SDK.  Ideally the sample code wouldn't use it (or even be able to include those dependencies)